purchase_packaging_level_qty/tests/test_purchase_packaging_level_qty.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2023 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import common
+
+
+class TestPurchasePackagingLevel(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.level1 = cls.env["product.packaging.level"].create(
+            {
+                "name": "Packaging Level 1 Test",
+                "code": "Pack1",
+                "sequence": 2,
+            }
+        )
+        cls.level2 = cls.env["product.packaging.level"].create(
+            {
+                "name": "Packaging Level 2 Test",
+                "code": "Pack2",
+                "sequence": 3,
+            }
+        )
+        cls.packaging1 = cls.env["product.packaging"].create(
+            {"name": "Packaging Test 1", "packaging_level_id": cls.level1.id, "qty": 4}
+        )
+        cls.packaging2 = cls.env["product.packaging"].create(
+            {"name": "Packaging Test 2", "packaging_level_id": cls.level2.id, "qty": 2}
+        )
+
+        cls.product = cls.env["product.product"].create(
+            {
+                "name": "Product Test",
+                "packaging_ids": [(6, 0, (cls.packaging1 | cls.packaging2).ids)],
+            }
+        )
+        cls.product2 = cls.env["product.product"].create(
+            {
+                "name": "Product Test2",
+            }
+        )
+
+        cls.purchase_order = cls.env["purchase.order"].create(
+            {
+                "company_id": cls.env.company.id,
+                "partner_id": cls.env.ref("base.res_partner_12").id,
+            }
+        )
+
+        cls.config = cls.env["res.config.settings"].create({})
+        cls.config.purchase_packaging_level_id = cls.level1
+        cls.config.execute()
+
+    def test_purchase_transport_packaging_level_enabled(self):
+        self.config.purchase_packaging_level_id = False
+        self.config.execute()
+        self.assertEqual(self.purchase_order.total_transport_qty, False)
+
+    # def test_purchase_packaging_level_qty(self):
```

Comment:
open question : 
is it useful to keep tests that are commented ? 

otherwise  LGTM

Reply:
it was not intentional. I uncomment thank you

Reply:
changed